<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8IFN1D6" failures="3" tests="3" name="Assertion Test" time="0.024" errors="0" timestamp="06 Feb 2020 18:06:43 GMT">
  <testcase classname="Testcase.AsserionExample" name="Fail" time="0.002">
    <failure type="java.lang.AssertionError" message="Failing the Test case">
      <![CDATA[java.lang.AssertionError: Failing the Test case
at Testcase.AsserionExample.Fail(AsserionExample.java:26)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Fail -->
  <testcase classname="Testcase.AsserionExample" name="fail" time="0.001">
    <failure type="java.lang.AssertionError" message="Testcase is failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Testcase is failed expected [true] but found [false]
at Testcase.AsserionExample.fail(AsserionExample.java:22)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- fail -->
  <testcase classname="Testcase.AsserionExample" name="validateTitles" time="0.009">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [yahoo.com] but found [gmail.com],
	Text box not present expected [true] but found [false],
	Test fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [yahoo.com] but found [gmail.com],
Text box not present expected [true] but found [false],
Test fail expected [true] but found [false]
at Testcase.AsserionExample.validateTitles(AsserionExample.java:17)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateTitles -->
</testsuite> <!-- Assertion Test -->
